src/stages/stagex/timeline.c

```diff
@@ -228,6 +228,73 @@ TASK(ngoner_fairy, { cmplx pos; }) {
 	e->move = move_linear(I);
 }
 
+static Boss *stagex_spawn_scuttle(cmplx pos0) {
+	Boss *scuttle = create_boss("Scutƫle", "scuttle", pos0);
+	boss_set_portrait(scuttle, "scuttle", NULL, "normal");
+	scuttle->shadowcolor = *RGBA(0.5, 0.0, 0.22, 1);
+	scuttle->glowcolor = *RGBA(0.30, 0.0, 0.12, 0);
+
+	return scuttle;
+}
+
+TASK(scuttle_appear, { cmplx pos; }) {
+	STAGE_BOOKMARK(midboss);
+	Boss *boss = global.boss = TASK_BIND(stagex_spawn_scuttle(ARGS.pos));
+
+	Attack *opening_attack = boss_add_attack(boss, AT_Normal, "Opening", 60, 40000, NULL, NULL);
+
+	boss_start_attack(boss, boss->attacks);
+	INVOKE_TASK(stagex_midboss_nonspell_1, ENT_BOX(boss), opening_attack);
+
+}
+
+TASK(scuttleproj_appear, NO_ARGS) {
+	Projectile *p = TASK_BIND(PROJECTILE(
+		.pos = VIEWPORT_W/2,
+		.proto = pp_soul,
+		.color = RGBA(0,0.2,1,0),
+		.move = move_towards(global.plr.pos, 0.015),
+		.flags = PFLAG_NOCLEAR | PFLAG_NOCOLLISION,
+		.max_viewport_dist = 10000, // how to do this properly?
```

Comment:
There is `PFLAG_NOAUTOREMOVE`. But be careful not to get a projectile stuck somewhere out of view forever.